tools/tools.go

```diff
@@ -9,6 +11,13 @@ import (
 	"github.com/symflower/eval-dev-quality/log"
 )
 
+// all holds all tools.
+var all []Tool
```

Comment:
Please keep code consistent. This registration code is totally different to the one in the other packages.

Reply:
@bauersimon please put it on your list to refactor the tools into their own packages, like we have with "model" and "provider".